dspace-api/src/main/java/org/dspace/eperson/GroupServiceImpl.java

```diff
@@ -654,6 +658,21 @@ public DSpaceObject getParentObject(Context context, Group group) throws SQLExce
                                 return collectionService.getParentObject(context, collection);
                             }
                         }
+                    } else {
+                        if (AuthorizeConfiguration.canCollectionAdminManagePolicies()) {
```

Comment:
This if is in another if that checks the following:
`                if (AuthorizeConfiguration.canCollectionAdminManageWorkflows()
                        || AuthorizeConfiguration.canCommunityAdminManageCollectionWorkflows()) {
`
Should that if not be expanded to include this one ?
Should it also not a include a check on AuthorizeConfiguration.canCommunityAdminManagePolicies() ?